runtime/themes/iroaseta.toml

```
@@ -0,0 +1,123 @@
+# Author: YardQuit
+
+# Syntax highlighting
+"attribute" = { fg = "yellow" }
+"comment" = { fg = "gray" }
+"constant" = { fg = "blue" }
+"constructor" = { fg = "orange" }
+"constant.builtin.boolean" = { fg = "blue" }
+"constant.character.escape" = { fg = "yellow" }
+"constant.character.numeric.integer" = { fg = "blue" }
+"constant.character.numeric.float" = { fg = "blue" }
+"function" = { fg = "purple" }
+"function.builtin" = { fg = "purple" }
+"function.method" = { fg = "green" }
+"function.macro" = { fg = "green" }
+"keyword" = { fg = "red" }
+"keyword.control.conditional" = { fg = "red" }
+"keyword.control.repeat" = { fg = "red" }
+"keyword.control.import" = { fg = "red" }
+"keyword.operator" = { fg = "blue" }
+"keyword.function" = { fg = "red" }
+"keyword.exception" = { fg = "red" }
+"keyword.return" = { fg = "red" }
+"operator" = { fg = "white" }
+"punctuation" = { fg = "white" }
+"punctuation.delimiter" = { fg = "white" }
+"punctuation.bracket" = { fg = "orange" }
+"punctuation.special" = { fg = "yellow" }
+"string" = { fg = "blue" }
+"string.special" = { fg = "blue" }
+"string.regexp" = { fg = "yellow" }
+"storage" = { fg = "red" }
+"storage.type" = { fg = "red" }
+"storage.modifier" = { fg = "yellow" }
```

```suggestion
"keyword.storage" = { fg = "red" }
"keyword.storage.type" = { fg = "red" }
"keyword.storage.modifier" = { fg = "yellow" }
```

these are nested under keyword